src/man/sssd-files.5.xml

```diff
@@ -51,6 +51,14 @@
                 <manvolnum>5</manvolnum>
             </citerefentry>.
         </para>
+	<para>
+	    SSSD never handles resolution of user/group "root". Also resolution of
+	    UID/GID 0 is not handled by SSSD. Such requests are passed to next
+	    provider (files).
+	</para>
+	<para>
+	    When SSSD is not running or responding, request is also passed to next provider.
```

Comment:
grammar police: the request, the next provider :-) Also, module is more precise than provider.

I would also make the sentence a little more detailed by saying that "nss_sss returns the code UNAVAIL which causes the request to be passed to the next module".